volmdlr/edges.py

```diff
@@ -2798,8 +2798,10 @@ def point_distance(self, point):
         return distance, volmdlr.Point3D(*point)
 
     def plane_projection2d(self, center, x, y):
-        return LineSegment2D(self.start.plane_projection2d(center, x, y),
-                             self.end.plane_projection2d(center, x, y))
+        start, end = self.start.plane_projection2d(center, x, y), self.end.plane_projection2d(center, x, y)
+        if start != end:
+            return LineSegment2D(start, end)
+        return None
```

Comment:
Don't you want to return a single Point2D when `start == end` ?

Reply:
Actually, if start == end > there will be a raise !

Reply:
Ah, sorry, you added this check (`start != end`), just to avoid the raise !